src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working.");
```

Comment:
The message should indicate that it is an `Excavator` that is working. Consider making the message more informative by specifying the type of machine in the message.